src/ApplicationManager.cpp

```
@@ -332,18 +338,27 @@ void ApplicationManager::handleSpoolDetected(const AppMessage& msg) {
         strncpy(lastDisplayedSpoolId, msg.payload.spoolDetected.spool_id, sizeof(lastDisplayedSpoolId) - 1);
         lastDisplayedSpoolId[sizeof(lastDisplayedSpoolId) - 1] = '\0';
-        lastDisplayedBlankId[0] = '\0';  // Clear so blank tag re-displays if swapped
+        lastDisplayedBlankId[0] = '\0';
 
-        char line1[17];
-        char line2[17];
-        snprintf(line1, sizeof(line1), "Type: %.10s", msg.payload.spoolDetected.material_name);
-        snprintf(line2, sizeof(line2), "Remain: %.0fg", msg.payload.spoolDetected.kg_remaining * 1000.0f);
-        lcdManager->updateScreen("**** Spool ****", "*** Scanned ***", line1, line2);
-    } else if (lcdManager) {
+        const auto& s = msg.payload.spoolDetected;
```

Clang (14.0.6)

[warning] variable name 's' is too short, expected at least 3 characters

(readability-identifier-length)